src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -384,6 +383,35 @@ void DirectoryService::ProcessDSBlockConsensusWhenDone(
 
     {
         lock_guard<mutex> g(m_mutexAllPOW1);
+
+        if (m_mode != IDLE)
+        {
+            //Copy POW1 to POW2
+            lock_guard<mutex> g2(m_mutexAllPOW2);
+            lock_guard<mutex> g3(m_mutexAllPoWConns);
+            m_allPoW2s.clear();
+
+            for (auto i : m_allPoW1s)
+            {
+                //Winner will become DS (leader), thus we should not put in POW2
+                if (m_allPoWConns[i.first] == winnerpeer)
+                {
+                    continue;
+                }
+
+                m_allPoW2s.insert(i);
+            }
+
+            //Add previous DS commitee (oldest one), because it back to normal node and should be collected
+            lock_guard<mutex> g4(m_mediator.m_mutexDSCommitteePubKeys);
+            lock_guard<mutex> g5(m_mediator.m_mutexDSCommitteeNetworkInfo);
+            m_allPoW2s.insert(make_pair(m_mediator.m_DSCommitteePubKeys.back(),
+                                        (boost::multiprecision::uint256_t)1));
```

Comment:
Try if it's possible to use `boost::multiprecision::uint256_t{1}`

Reply:
ok